nepalingo-web/src/lib/getNextWord.ts

```diff
@@ -0,0 +1,73 @@
+export const newariWords = [
+  "hello",
+  "call",
+  "can",
+  "do",
+  "how",
+  "I",
+  "my",
+  "what",
+  "where",
+  "a",
+  "do",
+  "not",
+  "for",
+  "from",
+  "fun",
+  "have",
+  "help",
+  "language",
+  "me",
+  "name",
+  "need",
+  "please",
+  "police",
+  "sick",
+  "speak",
+  "understand",
+  "well",
+  "you",
+  "your",
+  "salt",
+];
```

Comment:
Good job on creating an array of words for Newari language

Reply:
Binam dai had made the arary of words lol. Anyway, thank you 😂😂